projects/core/src/auth/user-auth/facade/auth.service.ts

```diff
@@ -179,10 +179,4 @@ export class AuthService {
   setLogoutProgress(progress: boolean): void {
     (this.logoutInProgress$ as BehaviorSubject<boolean>).next(progress);
   }
-
-  afterRedirectFromOidcLogin(): void {
```

Comment:
these logic will be done by https://github.com/SAP/spartacus/pull/19144/files#diff-f5970a8646b27e360071eded1b6a7614f2691bd271f329e371b4137cce332581L52